account_invoice_check_total/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2016 Acsone SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Account Invoice Check Total",
+    "summary": """
+        Check if the verification total is equal to the bill's total""",
+    "version": "13.0.1.0.0",
+    "website": "https://github.com/OCA/account-invoicing",
+    "author": "Acsone SA/NV, Odoo Community Association (OCA)",
```

Comment:
```suggestion
    "author": "Acsone SA/NV, Tecnativa, Odoo Community Association (OCA)",
```

Reply:
Not really. The migration itself is not very huge, so Tecnativa should only be contributor in this case.